index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div id="welcome">
```

Comment:
- [ ] @usamamaliq as previously requested kindly use `<a>` tag for this section, your welcome is supposed to be clickable and it's used in web design to take users to the homepage when clicked.